Tutorials/Lesson2/README.md

```diff
@@ -0,0 +1,33 @@
+In this tutorial, you will practice understanding counter-examples produced by Certora Prover. 
+As we discussed, the tool assumes all possible input values and states as a starting state. Some of them are infeasible, which means that there is no set of operations starting from the constructor that will lead to that state. 
+
+Start with [Ball Game](Ball.sol), implementing a ball game with four players. Player 1 passes the ball to Player 2; Player 2 passes back to Player 1. Player 3 and 4 passes to each other. The ball starts at Player 1. Let's prove that the ball can never reach player 4.
+
+run:  
+```sh
+certoraRun BallGame.sol --verify BallGame:BallGame.spec 
+```
+
+Understand the counter-example, fix the rule to avoid superfluous initial states. 
```

Comment:
Is this an instruction? Maybe put it as a numbered list or check boxes
 - [ ] run `certoraRun ...`
 - [ ] understand the counterexample
 - [ ] fix the rule to avoid sup...

If these are _not_ instructions maybe it should be reworded.